src/Core/Addon/Module/ModuleManager.php

```diff
@@ -221,9 +221,7 @@ protected function groupModulesByInstallationProgress()
             'to_update' => array(),
         );
 
-        /*
-         * @var \PrestaShop\PrestaShop\Adapter\Module\Module
-         */
+        // @var \PrestaShop\PrestaShop\Adapter\Module\Module
```

Comment:
```/** @var \PrestaShop\PrestaShop\Adapter\Module\Module $installedProduct ```

And I would even rename `$installedProducts` and `$installedProduct` into `$installedModules` and `$installedModule` because the current variables are not understandable

Reply:
Fixed